spec/services/proforma_service/convert_exercise_to_task_spec.rb

```diff
@@ -139,7 +161,7 @@
           usage_by_lms: 'display',
           visible: 'yes',
           binary: false,
-          internal_description: 'reference_implementation'
+          internal_description: nil
```

Comment:
Should we check additionally for the corresponding meta data with the file role?

Reply:
In a previous test, we ensure to update the metadata. Therefore this test doesn’t need to verify it.